lib/sequent/support/database.rb

```diff
@@ -30,17 +31,51 @@ def self.read_config(env)
       end
 
       def self.create!(db_config)
-        ActiveRecord::Base.establish_connection(db_config.merge(database: 'postgres'))
-        ActiveRecord::Base.connection.create_database(db_config[:database])
+        if Sequent.configuration.can_use_multiple_databases
+          ActiveRecord.legacy_connection_handling = false
```

Comment:
I see these couple of lines on multiple places, can we maybe  extract the into a single object?